schema/fleet_schema.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "name": "account_policy_data",
-    "examples": "Query the creation date of user accounts. You could also query the date of the last failed login attempt or password change.\n```\nSELECT strftime('%Y-%m-%d %H:%M:%S',creation_time,'unixepoch') AS creationdate FROM account_policy_data;\n```"
+    "examples": "Query the creation date of user accounts. You could also query the date of the last failed login attempt or password change.\n```\nSELECT strftime('%Y-%m-%d %H:%M:%S',creation_time,'unixepoch') AS creationdate FROM account_policy_data\n```\n\nSee each user's last password set date and number of failed logins since last successful login to detect any intrusion attempts.\n\n```SELECT \n    u.username\n    u.uid, \n    strftime('%Y-%m-%dT%H:%M:%S', a.password_last_set_time, 'unixepoch') AS password_last_set_time,\n    a.failed_login_count,\n    strftime('%Y-%m-%dT%H:%M:%S', a.failed_login_timestamp, 'unixepoch') AS failed_login_timestamp\nFROM account_policy_data AS a\nCROSS JOIN users AS u USING (uid)\nORDER BY password_last_set_time ASC\n;\n```"
```

Comment:
This is broken because of this part:

`\n\n```SELECT \n    u.username\n    u.uid, \n    strftime`

If you process it you will see it looks like this:

<img width="763" alt="image" src="https://user-images.githubusercontent.com/73836008/193864066-17bbe2d1-2314-4afe-bbbe-32c593c43466.png">


I will submit a commit to fix this plus my other comment above.

Reply:
thanks